src/views/Home/index.js

```diff
@@ -30,7 +28,9 @@ class Home extends Component {
   };
 
   createParty = async () => {
-    const { navigation, createParty } = this.props;
+    const { navigation, createParty, setModerator } = this.props;
+    const { username } = this.state;
+    setModerator(username);
```

Comment:
nitpicking, in this case I would just do `setModerator(this.state.username);`, since this is the only prop you need from the state. The destructuring on L32 feels unnecessary

Reply:
I agree with you but, that is done just to meet the linter rule `react/destructuring-assignment` and the prettier service will make the destructuring.

Reply:
oh, if that's part of the rules, fine. it was just a personal desire. 😅